tests/test_contracts/test_base.py

```diff
@@ -200,6 +209,57 @@ def test_scaffold():
         scaffold.get_state("ledger_api", "contract_address", **kwargs)
 
 
+def test_scaffolded_contract_method_call():
+    """Tests a contract method call."""
+
+    # Mock the CLI context
+    td = mkdtemp()
+
+    @dataclass
+    class AgentConfig:
+        author = "dummy_author"
+        contracts = ()
+        agent_name = "dummy_agent"
+
+    ctx = Context(cwd=td, verbosity="DEBUG", registry_path=td)
+    ctx.agent_config = AgentConfig()
+    ctx.config["to_local_registry"] = True
+    ctx.agent_config.directory = td
+
+    contract_name = "IUniswapV2ERC20"
+    contract_abi_path = Path("tests/test_contracts/IUniswapV2ERC20.json")
```

Comment:
again this won't work on Windows, use `os.path.join` or `pathlib.Path`

Reply:
It seems it works (it has not failed in CI) but I'll fix it anyway.